.github/workflows/docker.yml

```
@@ -62,6 +62,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+        # This should work without any of the project sources except compose.yaml
+        with:
+          sparse-checkout: |
+            compose.yaml
+          # Disable cone mode so we only grab a specific file
+          # https://github.com/actions/checkout?tab=readme-ov-file#fetch-only-a-single-file
+          # https://git-scm.com/docs/git-sparse-checkout
+          sparse-checkout-cone-mode: true
```

This doesn't match the linked example
```suggestion
          sparse-checkout-cone-mode: false
```